v7/german_slugify/german_slugify.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+from nikola.plugin_categories import SignalHandler
+from nikola import utils
+
+
+def _needs_german_slugifying_rules(lang, locale):
+    """Checks whether the given language name with corresponding locale needs German slugifying rules."""
+    if locale.split('_')[0] in {'de', 'gsw'}:  # German and Allemanic German
+        return True
+    if lang in {'de', 'deutsch', 'german',
```

Comment:
I removed `'deutsch'` and `'german'`. I left the others as I don't know of another way to distinguish between different german locales.